guidelines/principles/iconography/README.md

```diff
@@ -132,6 +164,14 @@ Icons should be of 2D
 Description here
```

Comment:
Add examples for the do's vs dont's in terms of perspective, roundness, or other visual properties that should be included